internal/driver/webhtml.go

```diff
@@ -53,90 +57,187 @@ button {
-  display: none;
-  position: fixed;
-  background-color: #ffffff;
-  border: 1px solid black;
-  box-shadow: 2px 2px 2px 0px #aaa;
-  top: 20px;
-  right: 20px;
-  z-index: 1;
+#closedetails {
+  display: inline;
+  float: right;
+  margin: 2px;
+}
+#home {
+  font-size: 14pt;
+  padding-left: 0.5em;
+  padding-right: 0.5em;
+  float: right;
 }
-.actionhdr {
-  background-color: #ddd;
+.menubar {
+  display: inline-block;
+  background-color: #f8f8f8;
+  border: 1px solid #ccc;
   width: 100%;
-  border-bottom: 1px solid black;
-  border-top: 1px solid black;
+}
+.menu-header {
+  position: relative;
+  display: inline-block;
+  padding: 2px 2px;
+  cursor: default;
   font-size: 14pt;
 }
-#actionbox > button {
+.menu {
+  display: none;
+  position: absolute;
+  background-color: #f8f8f8;
+  border: 1px solid #888;
+  box-shadow: 4px 4px 4px 0px rgba(0,0,0,0.2);
+  z-index: 1;
+  margin-top: 2px;
+  left: 0px;
+  min-width: 5em;
+}
+.menu hr {
+  background-color: #fff;
+  margin-top: 0px;
+  margin-bottom: 0px;
+}
+.menu button {
   display: block;
   width: 100%;
   margin: 0px;
   text-align: left;
-  padding-left: 0.5em;
+  padding-left: 2px;
   background-color: #fff;
-  border: none;
   font-size: 12pt;
+  border: none;
 }
-#actionbox > button:hover {
-  background-color: #ddd;
+.menu-header:hover {
+  background-color: #ccc;
 }
-#home {
-  font-size: 20pt;
-  padding-left: 0.5em;
-  padding-right: 0.5em;
+.menu-header:hover .menu {
+  display: block;
+}
+.menu button:hover {
+  background-color: #ccc;
+}
+#searchbox {
+  margin-left: 10pt;
+}
+#topcontainer {
+  width: 100%;
+  height: 100%;
+  max-height: 100%;
+  overflow: scroll;
+}
+#toptable {
+  border-spacing: 0px;
+}
+#toptable tr th {
+  border-bottom: 1px solid black;
+  text-align: right;
+  padding-left: 1em;
+}
+#toptable tr th:nth-child(6) {
+  text-align: left;
+}
+#toptable tr td {
+  padding-left: 1em;
+  font: monospace;
+  text-align: right;
+  white-space: nowrap;
+  cursor: default;
+}
+#toptable tr td:nth-child(6) {
+  text-align: left;
+}
+.hilite {
+  background-color: #ccf;
 }
 </style>
-</head>
-<body>
+{{end}}
 
-<button id="details">&#x25b7; Details</button>
+{{define "header"}}
 <div id="detailtext">
+<button id="closedetails">Close</button>
 {{range .Legend}}<div>{{.}}</div>{{end}}
 </div>
 
-<button id="reset">Reset</button>
-
-<span id="home">{{.Title}}</span>
-
-<input id="searchbox" type="text" placeholder="Search regexp" autocomplete="off" autocapitalize="none" size=40>
-
-<div id="page">
-
-<div id="errors">{{range .Errors}}<div>{{.}}</div>{{end}}</div>
+<div class="menubar">
+
+<div class="menu-header">
+View
+<div class="menu">
+{{if (ne .Type "top")}}
+  <button title="{{.Help.top}}" id="topbtn">Top</button>
+{{end}}
+{{if (ne .Type "dot")}}
+  <button title="{{.Help.graph}}" id="graphbtn">Graph</button>
+{{end}}
+<hr>
+<button title="{{.Help.details}}" id="details">Details</button>
+<button title="{{.Help.reset}}" id="reset">Reset</button>
```

Comment:
Would it be better to put "Reset" into the "Refine" menu, since its primary job appears to be throwing away the current focus, ignore, show and hide filters?